lighthouse/environment/src/lib.rs

```diff
@@ -182,149 +175,59 @@ impl<E: EthSpec> EnvironmentBuilder<E> {
         Ok(self)
     }
 
-    /// Sets a logger suitable for test usage.
-    pub fn test_logger(mut self) -> Result<Self, String> {
-        self.log = Some(test_logger());
-        Ok(self)
-    }
-
-    fn log_nothing(_: &mut dyn Write) -> IOResult<()> {
-        Ok(())
-    }
+    pub fn init_tracing(mut self, config: LoggerConfig) -> (Self, LoggingLayer, LoggingLayer) {
+        let mut log_path = config.path.unwrap();
```

Comment:
We'll need to remove this unwrap as well as Lighthouse will panic if there is no `log_path` set in the config. The file logging should be a completely optional part of the logging and should be able to be disabled if desired. If fact the `path` field of `LoggerConfig` is already an `Option<PathBuf>`.